### ISSUE TITLE
How to remove company portal from mac without sign in ?

### ISSUE THREAD

**Issue by gowrijsuria** [MicrosoftDocs/memdocs#1856]
[Enter feedback here]
I need to remove the company portal from my MacBook. However, I need to sign in to remove my device. This isn't possible because my temporary microsoft account has been deactivated after the completion of my internship. Kindly provide guidelines on how to go about this issue.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 515d32cb-c6b5-7487-347c-f4134291ce3d
* Version Independent ID: 1d0312aa-9068-19c4-9309-97606169ba56
* Content: [Unenroll device in Company Portal for macOS app - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/user-help/unenroll-your-device-from-intune-macos)
* Content Source: [memdocs/intune/user-help/unenroll-your-device-from-intune-macos.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/user-help/unenroll-your-device-from-intune-macos.md)
* Service: **microsoft-intune**
* Sub-service: **end-user**
* Product: ****
* Technology: ****
* GitHub Login: @lenewsad
* Microsoft Alias: **lanewsad**

**Comment by lenewsad** [comment]
Hi @gowrijsuria, thanks for reaching out to the Microsoft Intune docs team. I'm following up with the product team to help answer your question and will respond here when I hear back. Thanks!

**Comment by lenewsad** [comment]
Hi @gowrijsuria, here are the steps you need to take. 

1. Select the Apple menu > System Preferences > Profiles.
3. From the list of profiles, select **Management Profile**. 
4. Go down to the bottom of the list and you'll see a **plus (+)** and **minus (-)** sign.  Select the minus sign.
5. Follow the onscreen prompts to remove the management profile. 
6. When it's all done, drag the Company Portal app to **Trash** to unsintall it. 

Thanks so much for bringing this to our attention--we'll add these steps to the doc.

**Comment by gowrijsuria** [comment]
Thank you @lenewsad

**Comment by momusa23** [comment]
I am facing the same issue as above. There are no visible management profiles and I have uninstalled company portal and restarted the Mac several times. I also went into keychain access to delete the company portal related entries. The issue I have is that Intune password policy still applies to my device after apparent unenrollment in the MDM service. How can I get the Intune password policy removed?
